src/lib/fetchRequests.ts

```diff
@@ -48,6 +48,20 @@ export async function fetchSpecificSession(
 	});
 }
 
+export async function fetchClassroomSession(
```

Comment:
ah ah ah you didn't say the magic word.
duplicate kode:
![image](https://github.com/user-attachments/assets/f8c554be-b43b-4efa-b5f1-59d17eb18b34)

Reply:
men navnet på din er måske bedre